### PR TITLE
fix #531 time.secondsAndNanoseconds

### LINKED ISSUE #531: Time#get secondsAndNanoseconds bug
I've isolated an issue in Time where _nanoseconds is being modified when calling the getter 'secondsAndNanoseconds' in version 0.10.2 on MacOS Catalina. 

This example program prints out the _nanoseconds before and after calling secondsAndNanoseconds. I can see _nanoseconds change after calling secondsAndNanoseconds. If I comment out the call to secondsAndNanoseconds, _nanoseconds remains unchanged. Notice in the example output how time2 values of _nanosecond are a subset of the original time1 value.

**example output**
time1 1572996022175992874
time2 572996022
time1 1572996022925996911
time2 572996022
time1 1572996023677128755
time2 572996023
time1 1572996024427889376
time2 572996024
time1 1572996025178983458
time2 572996025
time1 1572996025932756150
time2 572996025

```javascript
const rcl = require('rclnodejs');

rcl.init().then(() => {
  const node = rcl.createNode('timer_ghost_node');
  const clock = new rcl.Clock();

  setInterval(() => {
    const now = clock.now();
    console.log('time1', now._nanoseconds);
    
    // the following getter is goofing up the _nanoseconds value of variable "now"
    // comment out this next line and _nanoseconds remains constant
    now.secondsAndNanoseconds;
    
    console.log('time2', now._nanoseconds);
  
  }, 750);

  rcl.spin(node);
});
```

### PR DESCRIPTION
**Pull request by wayneparrott**
Revised implementation of Time.secondsAndNanoseconds to avoid Time._nanoseconds being modified. Includes test.

### CODE REVIEW

File: test/test-time.js [hunk 1]
```diff
@@ -148,6 +148,10 @@ describe('rclnodejs Time/Clock testing', function() {
     assert.throws(() => {
       time.add(result);
     }, TypeError);
+
+    let nanos = time._nanoseconds;
+    time.secondsAndNanoseconds;
+    assert.strictEqual(time._nanoseconds,nanos);
```
Comment (minggangw) [inline test/test-time.js]: A space is needed after the comma here to get the jslint passed.

**Review by minggangw — COMMENTED** [review]
Thanks for your PR, almost done except some coding style issues, which failed the CIs. You can run it locally by `npm run test` or check results on the pull request page.

**Comment by wayneparrott** [comment]
Bummer! After making the small style change, 2 of 3 test suites fail for reason other than my change. Please advise.

**Comment by minggangw** [comment]
I checked the log and found that:

- CircleCI (macOS): caused by the wrong poco version, I have submitted an [issue](https://github.com/ros2/ros2/issues/813).
- AppVeyor (Windows10): I cannot reproduce this failure on Linux host (Sometimes, it seems that the nightly build package is not stable enough, I often meet some weird problems. I prefer to retrigger the CIs once a new ROS2 package is available.)

**Comment by minggangw** [comment]
All the binary packages used by the CIs are downloaded from https://ci.ros2.org/view/packaging/

**Pull request merged**